calculator/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+.container {
```

Comment:
This variant is possible to center an element.
But if you try to change the width of the browser you will see a problem case

Reply:
fixed it, for a better option